ecosystem/typescript/sdk_v2/src/bcs/serializer.ts

```diff
@@ -320,13 +327,11 @@ function checkNumberRange<T extends AnyNumber>(minValue: T, maxValue: T, message
   return (target: unknown, propertyKey: string, descriptor: PropertyDescriptor) => {
     const childFunction = descriptor.value;
     // eslint-disable-next-line no-param-reassign
-    descriptor.value = function deco(value: AnyNumber): Serializer {
-      const valueBigInt = BigInt(value.toString());
-      if (valueBigInt > BigInt(maxValue.toString()) || valueBigInt < BigInt(minValue.toString())) {
-        throw new Error(message || "Value is out of range");
-      }
+    descriptor.value = function deco(value: AnyNumber) {
+      validateNumberInRange(value, minValue, maxValue, message);
```

Comment:
what's `message`?

Reply:
whoops, will take that out, was supposed to be an error message for the function

Reply:
it's supposed to be a custom optional error message for the decorator, was from the original checkNumberRange sdk v1 function, but yeah we don't use it right now, remove?

Reply:
I mean if we don't use it....